source/administration/release-lifecycle.rst

```diff
@@ -1,67 +1,84 @@
 Release Lifecycle
 ========================
 
-What is an Release Lifecycle?
+What is a Release Lifecycle?
 ------------------------------------
-During each monthly release, Mattermost backports security and major bug fixes to the previous two monthly releases. Extended Support Releases receive backports for security fixes and major bug fixes for the length of their life cycle.  Once a release life cycle has ended, it will no longer receive the security or major bug backports. 
+During each monthly release, Mattermost backports security and major bug fixes to the previous two monthly releases. Extended Support Releases receive backports for security fixes and major bug fixes for the length of their life cycle.  Once a release life cycle has ended, it will no longer receive the security or major bug backports.
```

Comment:
> During each monthly release, Mattermost backports security and major bug fixes to the previous two monthly releases. 

Did this change to the previous 3 monthly releases recently?

Reply:
Yes, thank you, fixed.

Reply:
I think this one still needs to be updated to three.  See it updated in the other doc.